test/testdata/dsl/t_struct_default.rb.dsl-tree.exp

```diff
@@ -0,0 +1,35 @@
+class <emptyTree><<C <root>>> < ()
+  class <emptyTree>::<C Default><<C <todo sym>>> < (<emptyTree>::<C T>::<C Struct>)
+    ::T::Sig::WithoutRuntime.sig() do ||
+      <self>.params({:"foo" => <emptyTree>::<C Integer>}).void()
+    end
+
+    def initialize<<C <todo sym>>>(foo: = ::T.unsafe(nil), &<blk>)
```

Comment:
What do you think about duplicating the default argument into the initialize? That will have the added benefit that this becomes a type error:

```ruby
# typed: true
class A
  prop :foo, Integer, default: ''
end
```

https://sorbet.run/#%23%20typed%3A%20true%0Aclass%20A%0A%20%20prop%20%3Afoo%2C%20Integer%2C%20default%3A%20''%0Aend

Reply:
Oh nice. I didn't consider that there would be benefits to carting around the default expression but this shows that there are. I'll work on that.